src/App.js

```diff
@@ -1,12 +1,35 @@
-import React from 'react'
-import { data } from './mock'
+import React, { useEffect, useState } from 'react'
+import { Spin } from 'antd'
 import 'antd/dist/antd.css'
 import SequenceDiagram from './sequence-diagram/SequenceDiagram'
 
 function App() {
+  const [loading, setLoading] = useState(true)
+  const [sequenceDiagram, setSequenceDiagram] = useState(null)
+
+  useEffect(() => {
+    fetch('/mock.json')
+      .then((r) => {
+        return r.json()
+      })
+      .then((data) => {
+        setSequenceDiagram(data)
+      })
+      .finally(() => {
```

Comment:
we don't need this, right?

Reply:
what? the finally clause?

Reply:
yes. I mean, why do we need to add this delay?

Reply:
oh, sure, the delay is there while we're mocking the json otherwise the loader would just flick on the screen due to very low response latency